docker/ci/script/run.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash
 
-/usr/local/bin/docker-compose -p $1 run -v $1-gradle:/home/jenkins/.gradle -v $1-m2:/home/jenkins/.m2 --rm --user jenkins --entrypoint "$2" my-studies-builder
+USER_NAME="$(id -un)"
+USER_ID="$(id -u)"
+GROUP_NAME="$(id -gn)"
+GROUP_ID="$(id -g)"
+
+stop_and_remove_containers() {
+  /usr/local/bin/docker-compose -p $1 stop
+  /usr/local/bin/docker-compose -p $1 rm -f -v
+}
+
+stop_and_remove_containers $1
+/usr/local/bin/docker-compose build --build-arg USER_NAME="$USER_NAME" --build-arg USER_ID=$USER_ID --build-arg GROUP_NAME="$GROUP_NAME" --build-arg GROUP_ID=$GROUP_ID
+/usr/local/bin/docker-compose -p $1 run -v ~/.ssh/:/home/$USER_NAME/.ssh:ro -v ~/.gitconfig:/home/$USER_NAME/.gitconfig -v $1-gradle:/home/$USER_NAME/.gradle -v $1-m2:/home/$USER_NAME/.m2 --rm --user $USER_NAME --entrypoint "$2" my-studies-builder
```

Comment:
`.gitconfig`in voisi mountata read-onlynä

(Voisi myös rivittää argumentit omille riveilleen niin olisi helppolukuisempi)